### DISCUSSION TITLE
The looming demise of the 10x programmer

### DISCUSSION THREAD

**Discussion by cathycolliver** [therubyradar/submissions#152] [Submissions]
Why Ruby Radar audience will care: Justin Searls wrote an in-depth reflection about the 10x developer idea and how a lot of the current state goes back to a kind of one-time aberrant situation that led to a specific generation of programmers emerging that is very different from the next generation of programmers.

As someone else put it, food for thought even if you disagree!

Link: https://blog.testdouble.com/posts/2023-07-12-the-looming-demise-of-the-10x-developer/

**Comment by andrewmcodes** [comment]
Thanks so much for the submission! We will put it in today's issue 🙌
